dags/oup/oup_process_file.py

```diff
@@ -52,7 +53,7 @@ def enrich_file(enhanced_file):
 
     @task()
     def validate_record(enriched_file):
-        oup_validate_record(enriched_file)
+        return enriched_file and oup_validate_record(enriched_file)
```

Comment:
no, it will return the record:
https://github.com/cern-sis/workflows/blob/46158b9ec68c8ab5a1723154408235b4ecc2dc32/dags/oup/oup_process_file.py#L34-L37

Reply:
if enriched_files is false it will fail on validation and raise an error

Reply:
Yes which is the same with `return enriched_file and oup_validate_record(enriched_file)`

Reply:
yes, that's the check you were referring to, no? 
```
then why do we need to check for the enriched_file?
```

Reply:
I don't undrestand what is wrong here. Can you please gove me more details, what the problem is pointed here?